lib/publiccloud/azure.pm

```diff
@@ -384,13 +394,14 @@ sub create_image_definition {
     my $images = script_output("az sig image-definition list -g '$resource_group' -r '$gallery'");
     record_info("img-defs", "Existing image definitions:\n$images");
 
-    my $sku = get_var("PUBLIC_CLOUD_AZURE_SKU", $default_sku);
+    my $sku = get_var('PUBLIC_CLOUD_AZURE_SKU', $default_sku);
```

Comment:
please revert this line : 
- it is unrelated to current PR anyhow 
- you replacing double-quotes with single quotes in random place - it does not bring any value

Reply:
No sorry, before of that I created a function `sku()` that then I reverted to original, didn't note the quote format.

Reply:
Anyway, in this module single and double quotes are already used mixed in many get_var calls, without impacts, but I agree to avoid unuseful changes.

Reply:
done